cmd/add.rb

```diff
@@ -98,7 +104,16 @@ def add
 
                 # Add the formula/cask to the file if it hasn't already been added.
                 unless current_bundle_list.include?(brew.full_name) || (brew_name_resolves_to_full_name && current_bundle_list.include?(brew_name))
-                    file << "#{brewfile_prefix_type} #{quote_type}#{brew.full_name}#{quote_type}" << "\n"
+                    # Adapted from `BrewDumper.dump`:
+                    # https://github.com/Homebrew/homebrew-bundle/blob/master/lib/bundle/brew_dumper.rb#L59-L64
+                    brewline = if args.describe? && brew.desc
+                        puts "got env #{ENV.fetch('HOMEBREW_BUNDLE_DUMP_DESCRIBE', nil)}"
```

Comment:
Oops, didn't realize I actually pushed this debugging change.
- [ ] Probably should undo that before merging this.